slides/1-introduction.html

```diff
@@ -69,29 +77,101 @@ <h2><i class="fas fa-network-wired"></i> Internet</h2>
                     <textarea data-template>
                         ## <i class="fas fa-network-wired"></i> Domain Name System
 
-                        The <b>Domain Name System (DNS)</b> is a hierarchical and decentralized naming system (phone book) for computers connected to the Internet. 
-                        It translates domain names to IP addresses needed for locating and identifying computers.
+                        Hierarchical and decentralized implementation of a dictionary :
+
+                        <div class="centered">
+
+                          **Domain name ⟼ IP Address**
+
+                        </div>
 
                         <img alt="Networking" src="images/network_dns.svg" style="width: 65%;" />
 
-                        The DNS protocol uses <b>TCP</b> for Zone transfer and <b>UDP</b> for name queries.
+                        Uses TCP for zone transfer and UDP for name queries.
+
+                        Notes:
```

Comment:
Throughout the deck, I have rewritten some slides to be less wordy, and moved the text to a speaker note that will be visible to students. These notes sometimes go into more details than the original slide.